bazel-java-deps.bzl

```diff
@@ -41,7 +41,6 @@ def install_java_deps():
             "com.auth0:jwks-rsa:0.11.0",
             "com.chuusai:shapeless_{}:2.3.3".format(scala_major_version),
             "com.github.ben-manes.caffeine:caffeine:2.8.0",
-            "com.github.ghik:silencer-lib_{}:1.7.1".format(scala_version),
```

Comment:
Did you repin? I guess it won’t change much but I think I’d feel more confident if we at least bump the hash on any change to this file.

Reply:
I did. It's not a direct dependency but it's still a transitive dependency of the plugin, so the Maven pin files didn't change.